app/controllers/candidate_interface/course_choices/replace_choices/decision_controller.rb

```diff
@@ -20,6 +20,16 @@ def route_action
 
           if @pick_replacement_action.valid? && @pick_replacement_action.replacement_action == 'replace_location'
             redirect_to candidate_interface_replace_course_choice_location_path(@course_choice.id)
+          elsif @pick_replacement_action.valid? && @pick_replacement_action.replacement_action == 'replace_study_mode'
+            replacement_course_option_id = @course_choice.course_option.get_alternative_study_mode.id
+
+            redirect_to candidate_interface_confirm_replacement_course_choice_path(@course_choice.id, replacement_course_option_id)
+          elsif @pick_replacement_action.replacement_action == 'keep_choice'
+            if only_one_course_choice_needs_replacing?
+              redirect_to candidate_interface_application_complete_path
+            else
+              redirect_to candidate_interface_replace_course_choices_path
+            end
```

Comment:
Should there be a flash here? Nothing is actually being updated, but it might make sense to show one anyway.

Reply:
I think it's OK not to 👌